applications/HDF5Application/custom_io/hdf5_partitioned_model_part_io.cpp

```diff
@@ -20,6 +20,16 @@ PartitionedModelPartIO::PartitionedModelPartIO(File::Pointer pFile, std::string
     KRATOS_CATCH("");
 }
 
+PartitionedModelPartIO::PartitionedModelPartIO(File::Pointer pFile, std::string const& rPrefix, const bool IsUpdatedMeshIndicationFlagUsed, const Flags& rUpdatedMeshIndicationFlag)
+: ModelPartIO(pFile, rPrefix, IsUpdatedMeshIndicationFlagUsed, rUpdatedMeshIndicationFlag)
```

Comment:
if this goes in as is then please at least have one Ctor call the other to avoid code duplication (check how this is done in the ModelPart)